droidlet/tools/hitl/nsp_retrain/README.md

```diff
@@ -0,0 +1,38 @@
+## How to run the pipeline end-to-end
+
+### Step 1: set up environment variables for credentials (e.g. AWS, Mturk, Cloudflare)
+
+Here's a list of environment variables you should set before you run the pipeline:
+1. MEPHISTO_AWS_ACCESS_KEY_ID
+2. MEPHISTO_AWS_SECRET_ACCESS_KEY
+3. MEPHISTO_REQUESTER
+4. AWS_ACCESS_KEY_ID
+5. AWS_SECRET_ACCESS_KEY
+6. AWS_DEFAULT_REGION
+7. CLOUDFLARE_TOKEN
+8. CLOUDFLARE_ZONE_ID
+9. CLOUDFLARE_EMAIL
+10. MTURK_AWS_ACCESS_KEY_ID
+11. MTURK_AWS_SECRET_ACCESS_KEY
+
+1-2 is the AWS credentials you provided to Mephisto to create HITs  
+3 is the name of Mephisto requester you registered  
+4-6 is the AWS info we use to spin up ECS instances for hosting dashboard, you should provide one with ECS,ECR access permission  
+7-9 is the cloudflare info which we use to register domain names for dashboard urls  
+10-11 is the AWS credentials we use to spin up annotation jobs  
```

Comment:
Without prior knowledge, I found the distinction between 1-2 and 10-11 to be confusing because both are MTruk credentials.  You could specify that the former is for interaction jobs and/or maybe write a line about in what cases the two sets of MTurk credentials would be different.

Reply:
thanks for bringing it up. I added more detailed explanation in README about those credentials.